tktermwidget/style.py

```diff
@@ -0,0 +1,27 @@
+"""Styles for terminal"""
+
+Default = { # Style for normal tkterminalwidget
+    "background": "#2B2B2B",
+    "insertbackground": "#DCDCDC",
+    "selectbackground": "#b4b3b3",
+    "selectforeground": "#e6e6e6",
+    "foreground": "#cccccc",
+}
+
+Powershell = { # Style for powershell
+    "background": "#012456",
+    "insertbackground": "#eeedf0",
+    "selectbackground": "#fedba9",
+    "selectforeground": "#11120f",
+    "foreground": "#cccccc",
+}
+
+Command = { # Style for normal "cmd.exe"
+    "background": "#000000",
+    "insertbackground": "#f2f2f2",
+    "selectbackground": "#f3f3f3",
+    "selectforeground": "#000000",
+    "foreground": "#f2f2f2",
+}
+
+# TODO: add a user custom style function later...
```

Comment:
I'd make a small guide on how to make the dicts.

Reply:
We don't need to do it, we can write a gui to let use choose their custom color and we write it to a json file with the format we write. (We don't need to write a guide.)

Reply:
I wouldn't make the gui usable when using the terminal though as it might cause issues for some people ;).

Reply:
That all right, they can use ```write_style``` to write their own style without ui